src/Tizen.Security.DevicePolicyManager/Tizen.Security.DevicePolicyManager/PasswordPolicy.cs

```diff
@@ -0,0 +1,190 @@
+﻿/*
+ *  Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ */
+
+using System;
+
+namespace Tizen.Security.DevicePolicyManager
+{
+    /// <summary>
+    /// The PasswordPolicy provides methods to manage password policies.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class PasswordPolicy : DevicePolicy
+    {
+        internal PasswordPolicy(DevicePolicyManager dpm) : base(dpm)
+        {
+        }
+
+        /// <summary>
+        /// Gets the number of days password expires.
+        /// </summary>
+        /// <value>Number of days after which the password expires. The default value is maximum of int.</value>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/dpm.password</privilege>
+        /// <privlevel>partner</privlevel>
+        public int ExpiresDay
```

Comment:
Please consider changing the name to "DaysToExpiration".

Reply:
I changed the name to "DaysToExpiration".